js/tree.js

```diff
@@ -89,28 +93,59 @@
 
 		populate: function populate($el, isBackgroundProcess) {
 			var self = this;
+
+			// populate was initiated based on clicking overflow link
+			var isOverflow = $el.hasClass('tree-overflow');
+
 			var $parent = ($el.hasClass('tree')) ? $el : $el.parent();
-			var loader = $parent.find('.tree-loader:eq(0)');
+			var atRoot = $parent.hasClass('tree');
+
+			if(isOverflow && !atRoot){
+				$parent = $parent.parent();
+			}
+
 			var treeData = $parent.data();
+			// expose overflow data to datasource so it can be responded to appropriately.
+			if(isOverflow){
+				treeData.overflow = $el.data();
+			}
+
 			isBackgroundProcess = isBackgroundProcess || false;	// no user affordance needed (ex.- "loading")
 
+			if(isOverflow){
+				if(atRoot){
+					// the loader at the root level needs to continually replace the overflow trigger
+					// otherwise, when loader is shown below, it will be the loader for the last folder
+					// in the tree, instead of the loader at the root level.
+					$el.replaceWith($parent.find('> .tree-loader').remove());
+				}else{
+					$el.remove();
+				}
+			}
+
+			var $loader = $parent.find('.tree-loader:last');
+
 			if (isBackgroundProcess === false) {
-				loader.removeClass('hide hidden'); // hide is deprecated
+				$loader.removeClass('hide hidden'); // hide is deprecated
```

Comment:
Change this comment to:
`//in jQuery >= v3.0 hide and show are changing behaviors`